package.json

```diff
@@ -105,23 +105,23 @@
     "grunt-webpack": "^6.0.0",
     "jsdom": "^22.1.0",
     "jspdf": "^2.5.1",
-    "npm": "^9.8.1",
+    "npm": "^9.9.0",
     "opener": "^1.5.2",
     "opentype.js": "^1.3.4",
-    "prettier": "^3.0.1",
-    "puppeteer": "^21.0.1",
-    "qunit": "^2.19.4",
+    "prettier": "^3.0.3",
+    "puppeteer": "^21.3.8",
+    "qunit": "^2.20.0",
     "recursive-copy": "^2.0.14",
-    "release-it": "^16.1.3",
-    "sharp": "^0.32.4",
+    "release-it": "^16.2.1",
+    "sharp": "^0.32.6",
     "string-replace-loader": "^3.1.0",
-    "svg2pdf.js": "^2.2.1",
+    "svg2pdf.js": "^2.2.2",
     "terser-webpack-plugin": "^5.3.9",
-    "ts-loader": "^9.4.4",
+    "ts-loader": "^9.5.0",
     "tsc-watch": "^6.0.4",
     "typedoc": "^0.24.8",
-    "typescript": "^5.1.6",
-    "webpack": "^5.88.2"
+    "typescript": "^5.2.2",
```

Comment:
typedoc is no longer required. Are you planning to set up vexflow-docs?

Reply:
What's the URL of the github pages? Does the 4.x release have its own separate github pages for the API docs?